cpp/daal/src/algorithms/dtrees/forest/regression/df_regression_train_dense_default_impl.i

```diff
@@ -51,12 +51,12 @@ using namespace dtrees::training::internal;
 //where mean, var are mean and mse for s,
 //where mean1, var1 are mean and mse for s1
 template <typename algorithmFPType, CpuType cpu>
-void subtractImpurity(algorithmFPType var, algorithmFPType mean, algorithmFPType var1, algorithmFPType mean1, size_t n1, algorithmFPType & var2,
-                      algorithmFPType & mean2, size_t n2)
+void subtractImpurity(algorithmFPType var, algorithmFPType mean, algorithmFPType var1, algorithmFPType mean1, algorithmFPType n1,
```

Comment:
Looks like n isn't a number of something. If so, consider better name

Reply:
fixed